eclipse-che/che-operator
# feat: enable nativeUserMode on OpenShift with devworkspace by default

Signed-off-by: Michal Vala <mvala@redhat.com>



### What does this PR do?
When che-operator is deploying on OpenShift with DevWorkspaces, enable `nativeUserMode` by default.


### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/20203


### How to test this PR?

1. `deploy with chectl server:deploy --platform=openshift --installer=operator --workspace-engine=dev-workspace --che-operator-image=quay.io/mvala/che-operator:gh20203-nativeUserModeEnable` (chectl will get stuck on Keycloak, but Che will deploy fine)
2. `nativeUserMode` should be enabled by default


### PR Checklist
          
[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
